doc/services/rtio/index.rst

```diff
@@ -216,13 +216,3 @@ API Reference
 *************
 
 .. doxygengroup:: rtio
-
-MPSC Lock-free Queue API
-========================
-
-.. doxygengroup:: rtio_mpsc
-
-SPSC Lock-free Queue API
-========================
-
-.. doxygengroup:: rtio_spsc
```

Comment:
Removing these categories from RTIO pages help with understanding RTIO: MPSC and SPSC were looking like part of the API, but are only used internally rather than two variants of RTIO.